public_website/tests/test_index.py

```diff
@@ -0,0 +1,17 @@
+from django.test import TestCase
+from django.urls import resolve, reverse
+from public_website import views
+
+class TestIndex(TestCase):
+
+    def test_index_url_calls_right_view(self):
+        resolved = resolve(reverse('index'))
```

Comment:
Tu peux aussi mettre
```suggestion
        resolved = resolve("/")
```